sphinx_needs/layout.py

```diff
@@ -130,9 +133,14 @@ def build_need(layout, node, app: Sphinx, style=None, fromdocname: Optional[str]
 
     lh = LayoutHandler(app, need_data, need_layout, node, style, fromdocname)
     new_need_node = lh.get_need_table()
+    node_container.append(new_need_node)
+
+    container_id = "SNCB-" + str(uuid.uuid4())[:32]
```

Comment:
I thin we do not need such long ids.
It only must be unique on the same page, or not?
So 4 randoms chars should be enough.

Reply:
Okay noted.